README.pt-br.md

```diff
@@ -602,11 +620,207 @@ ObxValue(
+
+// a igualdade funciona com o Rx e o valor, mas hashCode é sempre obtido do valor 
+final number = 12.obs;
+print( number == 12 ); // prints > true
+
+/// Rx Models personalizados:
+
+// toJson(), toString() são transferidos para o filho, para que você possa implementar 
+// override neles e imprimir o observável diretamente.
+
+class User {
+    String name, last;
+    int age;
+    User({this.name, this.last, this.age});
+
+    @override
+    String toString() => '$name $last, $age years old';
+}
+
+final user = User(name: 'John', last: 'Doe', age: 33).obs;
+
+// `user` é "reativo", mas as propriedades dentro NÃO SÃO!
+// Então, se mudarmos alguma variável dentro dele:
+user.value.name = 'Roi';
+// O widget não vai reconstruir!,
+// `Rx` não tem nenhuma notificação quando você muda algo dentro do usuário.
+// Portanto, para classes personalizadas, precisamos "notificar" manualmente a mudança.
+user.refresh();
+
+// ou podemos usar o método `update()`!
+user.update((value){
+  value.name='Roi';
+});
+
+print( user );
+```
+
+#### GetView
+
+Eu amo este Widget, é tão simples, mas tão útil!
+
+É um Widget `const Stateless` que tem um getter `controller` registrado para Controller, só isso.
+
+```dart
+class AwesomeController extends GetxController {
+  final String title = 'My Awesome View';
+}
+
+// SEMPRE lembre de passar o `Type` que você usou para registrar seu controlador!
+class AwesomeView extends GetView<AwesomeController> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.all(20),
+      child: Text( controller.title ), // apenas chame `controller.something`
+    );
+  }
+}
+```
+
+#### GetWidget
+
+A maioria das pessoas não tem ideia sobre este widget, ou confunde totalmente o uso dele. 
+O caso de uso é muito raro, mas muito específico: Ele armazena em `cache` um Controller. 
+Por causa do _cache_, não pode ser um `const Stateless`.
+
+> Então, quando você precisa armazenar em "cache" um Controller?
+
+Se você usar, uma outra característica "não tão comum" de **GetX™**: `Get.create()`.
+
+`Get.create(()=>Controller())` irá gerar um novo `Controller` cada vez que você chamar
+`Get.find<Controller>()`,
+
+É aí que `GetWidget` brilha... já que você pode usá-lo, por exemplo, 
+para manter uma lista de itens Todo. Portanto, se o widget for "reconstruído", ele manterá a mesma instância do controlador.
+
+#### GetxService
+
+Esta classe é como um `GetxController`, ele compartilha o mesmo ciclo de vida ( `onInit()`, `onReady()`, `onClose()`). 
+Mas não tem "lógica" dentro dele. Ele apenas notifica o sistema de injeção de dependência do GetX™ de que esta subclasse
+**não pode** ser removida da memória.
+
+Portanto, é muito útil manter seus "Services" sempre acessíveis e ativos com `Get.find()`. Como: 
+`ApiService`, `StorageService`, `CacheService`.
+
+```dart
+Future<void> main() async {
+  await initServices(); /// Aguarda a inicialização dos Services.
+  runApp(SomeApp());
+}
```

Comment:
Pera, não entendi isso também
O que uma classe que extenda o GetxService faz de diferente?
pra poder instanciar classes que vão ser permanentes no ciclo de vida do app, o certo não é usar Bindings junto com `get.put(Class(), permanent: true)`?

e se eu vou executar um método de uma classe GetXService que tem vários get.put pra cada dependência, então o que essa classe tá fazendo de diferente?
A única coisa que eu percebi que é ela não é ligada a nenhuma rota, mas do jeito que tá escrito aqui parece que eu posso simplesmente criar uma classe normal com um método estático que roda os `Get.put()`

Reply:
@Nipodemos isso é bem novo e realmente ainda não consegui estudar para entender os benefícios, mas pelo que li é diferente dos Bindings, pois não são controllers e sim services que eu nem precisaria injetar como dependência, mas seriam responsáveis por fazer algumas configurações iniciais, inclusive iniciar alguns Controllers.
Nessa parte que destacou eu só fiz a tradução fiel da documentação em inglês (readme)

Reply:
entendi, bom acho que a melhor coisa então é perguntar pro autor 😅 
@roipeker if I do remember correctly you made these docs about GetxServices right? Would you mind explaining to me what is this for and what the advantage of using it instead of a normal class?
ou o @jonataslaw também, se puderem me esclarecer isso por favor

Reply:
GetxService não permite, por exemplo que você atualize um GetBuilder com o update, porque ele não tem update, não tem listener, não tem absolutamente nada de lógica dos controllers. 
Ele não é um Controller, e ele não pode ser acessado diretamente pela view. 
Ele serve para criar services permanentes (leia services com o mesmo sendo do Service no android nativo, que não morrem como as activitys). 
GetxServices são essenciais para serviços como Database, Auth, e qualquer outro serviço oponível (que se manterá sobre vários controllers), e eles não são excluídos a menos que você o exclua de forma forçada com um `Get.delete<Service>(force:true);`
E ele tem ciclo de vida, mas que diferente dos controllers, não são chamados em uma rota. Uma vez colocado na memória, ele torna-se permanente, a menos que alguém o remova explicitamente. Services podem ser carregados lazyLoad, o que diferencia ele de um GetxController com "permanent:true", que é iniciado desde o inicio.

Reply:
@jonataslaw, você acha que devo fazer alguma alteração na documentação pt-br nessa parte?